src/sentry/options/defaults.py

```diff
@@ -374,3 +374,5 @@
 # This is to allow gradual rollout of metrics collection for symbolication requests and can be
 # removed once it is fully rolled out.
 register("symbolicate-event.low-priority.metrics.submission-rate", default=0.0)
+
+register("performance:suspect-spans-ingestion-projects", default={})
```

Comment:
Doesn't look like we have any conventions for this, but I think you're right that it might fit better in getsentry. Though for this PR, I'm going to keep it here as this is not permanent and will be removed when we can take the feature flag off.